version.py

```diff
@@ -14,7 +14,7 @@
 ZULIP_MERGE_BASE = lines.pop(0).strip()
 
 LATEST_MAJOR_VERSION = "9.0"
-LATEST_RELEASE_VERSION = "9.1"
+LATEST_RELEASE_VERSION = "9.2"
 LATEST_RELEASE_ANNOUNCEMENT = "https://blog.zulip.com/2023/12/15/zulip-8-0-released/"
```

Comment:
Seems we forgot to update this to https://blog.zulip.com/2024/07/25/zulip-9-0-released/.

Reply:
See also https://chat.zulip.org/#narrow/stream/438-release-management/topic/LATEST_RELEASE_ANNOUNCEMENT/near/1902649

Reply:
Yeah, probably we should just delete it at this point.